pkg/controller/controller.go

```diff
@@ -433,11 +446,29 @@ func (ctrl *Controller) syncKubervisorService(bc *api.KubervisorService) (bool,
 	return false, nil
 }
 
-func updateGauge(name string, status api.PodCountStatus) {
-	kubervisorGauges.WithLabelValues(name, "managed").Set(float64(status.NbPodsManaged))
-	kubervisorGauges.WithLabelValues(name, "breaked").Set(float64(status.NbPodsBreaked))
-	kubervisorGauges.WithLabelValues(name, "paused").Set(float64(status.NbPodsPaused))
-	kubervisorGauges.WithLabelValues(name, "unknown").Set(float64(status.NbPodsUnknown))
+func updateGauge(name string, namespace string, status api.PodCountStatus) {
+	kubervisorGauges.WithLabelValues(name, namespace, "managed").Set(float64(status.NbPodsManaged))
+	kubervisorGauges.WithLabelValues(name, namespace, "breaked").Set(float64(status.NbPodsBreaked))
+	kubervisorGauges.WithLabelValues(name, namespace, "paused").Set(float64(status.NbPodsPaused))
+	kubervisorGauges.WithLabelValues(name, namespace, "unknown").Set(float64(status.NbPodsUnknown))
+}
+
+func deleteGauge(name, namespace string) {
+	kubervisorGauges.DeleteLabelValues(name, namespace, "managed")
+	kubervisorGauges.DeleteLabelValues(name, namespace, "breaked")
+	kubervisorGauges.DeleteLabelValues(name, namespace, "paused")
+	kubervisorGauges.DeleteLabelValues(name, namespace, "unknown")
+}
+
+func (ctrl *Controller) clearItem(name, namespace string) {
+	key := fmt.Sprintf("%s/%s", name, namespace)
```

Comment:
There is something suspicious in the formating of the key:
https://github.com/AmadeusITGroup/kubervisor/pull/27/files#diff-3c862eb54a8e0e161b534e0c67e5379eR356
and
https://github.com/AmadeusITGroup/kubervisor/pull/27/files#diff-3c862eb54a8e0e161b534e0c67e5379eR372

We should create a Key method with explicit parameters name to avoid confusion.

Reply:
Yes, I will create a dedicated method